rules/network/command_and_control_cobalt_strike_beacon.toml

```
@@ -40,7 +40,9 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset: (network_traffic.tls or network_traffic.http) AND destination.domain:/[a-z]{3}.stage.[0-9]{8}\..*/
+((event.category: (network or network_traffic) and type: (tls or http) and network.transport: tcp)
```

```suggestion
((event.category: (network or network_traffic) and type: (tls or http))
```

Confirmed via schema where `network.type` is network layer, `type` is "transaction" specific.